Content.Server/AI/Utility/AiLogic/Civilian.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Server.AI.Utility.BehaviorSets;
+using JetBrains.Annotations;
+using Robust.Server.AI;
+
+namespace Content.Server.AI.Utility.AiLogic
+{
+    [AiLogicProcessor("Civilian")]
+    [UsedImplicitly]
+    public sealed class Civilian : UtilityAi
+    {
+        public override void Setup()
+        {
+            base.Setup();
+            BehaviorSets.Add(typeof(ClothingBehaviorSet), new ClothingBehaviorSet(SelfEntity));
```

Comment:
you could add a method to avoid specifying the type twice in the signature, like so:

```csharp
public void AddBehaviorSet<T>(T set) where T : BehaviorSet 
{
    BehaviorSets.Add(typeof(T), set);
}
```
Which lets you instead do:
```csharp
AddBehaviorSet(new ClothingBehaviourSet());
```
Just feels more ergonomic.


This does prevent you from differing the type used as a key and the type used as a value, like so:
```csharp
BehaviorSets.Add(typeof(ISomeInterface), new ConcreteType());
BehaviorSets.Add(typeof(SomeTypeA), new SomeTypeB());
```
But I'm not sure we even want that? (and you could always leave the dictionary public to keep it possible.

Reply:
Definitely better, thanks Remie. I also added an arg so the sorting can be ignored (a minor optimisation as it should only be ignored during initialization anyway).